tools/cldr-apps/js/src/esm/cldrEvent.js

```diff
@@ -416,6 +416,8 @@ function unpackMenuSideBar(json) {
     toggleOverlay();
     const url = $(this).data("url");
     if (url === "dashboard") {
+      // Note: setCurrentSpecial("general") is dubious here; it doesn't cause
+      // the "general" page to be loaded; it doesn't hide whatever else was displayed.
```

Comment:
that makes sense. Eventually we might investigate what happens when choose Dashboard from left sidebar under unusual circumstances -- outside scope of this ticket... the specials "locales" and "general" could both use some freshening up, and also the open/closed state of dashboard should be remembered so that if you leave the Page table view while dash is open, then come back to it, the dash gets re-opened automatically -- similar to what this ticket accomplishes for info panel